resources/idlharness.js

```diff
@@ -1040,7 +1040,7 @@ IdlInterface.prototype.test_self = function()
 
         if (this.is_callback()) {
             assert_false("prototype" in self[this.name],
-                         this.name + ' should not have a "prototype" property');
+                         this.name + ' should have a "prototype" property');
```

Comment:
It's the same as the previous case IMO: we're saying "foo should have a 'prototype' property: expected false got true" instead of "foo should not have a 'prototype' property: expected false got true" which looks like "expected false to 'should not have a 'prototype' property' aka 'expected true to 'should have a 'prototype property''.